lib/rules/operator-linebreak.js

```diff
@@ -92,14 +114,33 @@ module.exports = function(context) {
         "AssignmentExpression": validateBinaryExpression,
         "VariableDeclarator": function(node) {
             if (node.init) {
-                validateBinaryExpression(node);
+                validateNode(node, node.id);
             }
+        },
+        "ConditionalExpression": function(node) {
+            validateNode(node, node.test);
+            validateNode(node, node.consequent);
         }
     };
 };
 
 module.exports.schema = [
     {
-        "enum": ["after", "before", "none"]
+        "enum": ["after", "before", "none", null]
+    },
+    {
+        "type": "object",
+        "properties": {
+            "overrides": {
+                "type": "object",
+                "properties": {
+                    "anyOf": {
+                        "type": "string",
```

Comment:
Ideally we would also enforce a valid operator as the property name but I couldn't find an easy way to do that with JSON-schema.